frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -526,6 +546,13 @@ class NativeQueryEditor extends Component {
 
 export default _.compose(
   ExplicitSize(),
+  Questions.loadList({
+    query: (state, props) => ({
+      questionId: props.question.id(),
+      endpoint: "referenced-cards",
+    }),
```

Comment:
Perhaps `endpoint` isn't the best name for this, as the query is generally contains no information about the endpoints for a given query. But I'm not sure this indirection is necessary or desired, actually. We could start adopting this as a general pattern.